### PR TITLE
Update README.md with possible projects

### PR DESCRIPTION
**Pull request by trontrytel**
This adds to the readme:
- a list of next development projects for and around the CloudMicrophysics.jl package
-  some basic installation instructions for users and developers.

I'm hoping this could be useful for students when looking for some projects to contribute to. Let me know what you think and how I should improve the readme

### CODE REVIEW

**Comment by trontrytel** [comment]
Fixes https://github.com/CliMA/CloudMicrophysics.jl/issues/33

File: README.md [hunk 1]
```diff
@@ -23,3 +23,91 @@ A package containing a library of cloud microphysics parameterizations
...
+
+The CloudMicrophysics.jl package is being actively developed
+  and welcomes contributions and feedback.
+There is a variety of projects big and small that are available to take up as
+  fun research projects for interested students and other contributors.
+Below is a list of possible examples,
+  but other suggestions and ideas are always welcome!
+
+- The CloudMicrophysics.jl package contains an aerosol activation parameterization
+  that is an implementation of [Abdul-Razzak and Ghan 2000](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/1999JD901161)
```
Comment (szy21) [inline README.md:82]:
```suggestion
  that is an implementation of [Abdul-Razzak and Ghan 2000](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/1999JD901161).
```

File: README.md [hunk 2]
```diff
@@ -23,3 +23,91 @@ A package containing a library of cloud microphysics parameterizations
...
+- Adding a 2-moment microphysics scheme for warm rain formation to the CloudMicrophysics.jl package.
+  This is a priority for us and the next development goal.
+  Contributions in coding, testing, (re)deriving the parameterization equations
+  and discussing microphysics parameterization assumptions are welcome!
+
+- Adding the [P3 scheme](https://journals.ametsoc.org/view/journals/atsc/72/1/jas-d-14-0065.1.xml)
+  for ice phase microphysics.
+  This is a next development goal after we implement the 2-moment scheme.
+
+- Adding an aerosol model and coupling it with the aerosol activation
```
Comment (szy21) [inline README.md:109]: Just a comment. I don't know whether we should have the aerosol model in `CloudMicrophysics`? But we can leave it here for now.

**Review by szy21 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (claresinger) [inline README.md:82]:
```suggestion
- The CloudMicrophysics.jl package contains an aerosol activation parameterization
  that is an implementation of [Abdul-Razzak and Ghan 2000](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/1999JD901161).
```

File: README.md [hunk 3]
```diff
@@ -23,3 +23,91 @@ A package containing a library of cloud microphysics parameterizations
...
+  as our high-resolution benchmark.
+  PySDM is a package for simulating the dynamics of population of particles
+  and is based on the [Super-Droplet algorithm](https://rmets.onlinelibrary.wiley.com/doi/abs/10.1002/qj.441).
+  Possible tasks in this project would include testing the aerosol activation parameterization
+  against PySDM in an adiabatic parcel setup, or testing the 1-moment
+  microphysics parameterization against PySDM in a 1-dimensional or
+  2-dimensional prescribed flow setup.
+  This could be extended further into a calibration exercise using the
+  [EnsembleKalmanProcesses.jl](https://github.com/CliMA/EnsembleKalmanProcesses.jl) package.
+
```
Comment (claresinger) [inline README.md:99]:
```suggestion
An example of this calibration pipeline can be seen in the documentation [here](https://clima.github.io/EnsembleKalmanProcesses.jl/dev/examples/Cloudy_example/) with an example of calibrating parameters from [Cloudy.jl](https://github.com/CliMA/Cloudy.jl).
```

File: README.md [hunk 4]
```diff
@@ -23,3 +23,91 @@ A package containing a library of cloud microphysics parameterizations
...
+- Adding a 2-moment microphysics scheme for warm rain formation to the CloudMicrophysics.jl package.
+  This is a priority for us and the next development goal.
+  Contributions in coding, testing, (re)deriving the parameterization equations
+  and discussing microphysics parameterization assumptions are welcome!
+
+- Adding the [P3 scheme](https://journals.ametsoc.org/view/journals/atsc/72/1/jas-d-14-0065.1.xml)
+  for ice phase microphysics.
+  This is a next development goal after we implement the 2-moment scheme.
```
Comment (claresinger) [inline README.md:107]:
```suggestion
  This is a next development goal after we implement the 2-moment scheme - and also a big task.
```

**Review by claresinger — APPROVED** [review]
Great! This should be very helpful for future students joining the team.

File: README.md (hunk 2 above)
Comment (trontrytel) [reply README.md:109]: Ha! Yeah. We might want to have it as separate repo, I agree. But can keep it in the tasks list here, for now.

**Review by charleskawczynski — COMMENTED** [review]
LGTM.

It might be nice to point to good Julia documentation/resources on package management since these things can be a bit tricky for newcomers.

File: README.md [hunk 5]
```diff
@@ -23,3 +23,94 @@ A package containing a library of cloud microphysics parameterizations
...
+
+## Installation and running instructions
+
+CloudMicrophysics.jl is a Julia registered package.
+It depends on a couple of standard Julia packages as well as
+  the [Thermodynamics.jl](https://github.com/CliMA/Thermodynamics.jl) and
+  [CLIMAParameters.jl](https://github.com/CliMA/CLIMAParameters.jl)
+  (two Julia packages developed at [CliMA](https://github.com/CliMA)).
+See the [Project.toml](https://github.com/CliMA/CloudMicrophysics.jl/blob/main/Project.toml)
+  for details.
```
Comment (charleskawczynski) [inline README.md:35]:
```suggestion
See the [Project.toml](https://github.com/CliMA/CloudMicrophysics.jl/blob/main/Project.toml)
  for a full list of CloudMicrophysics.jl's dependencies.
```

File: README.md [hunk 6]
```diff
@@ -23,3 +23,94 @@ A package containing a library of cloud microphysics parameterizations
...
+
+When using the CloudMicrophysics.jl inside your own project,
+  the easiest way to obtain the latest stable version of the package
+  and it's dependencies is to use the Julia built-in package manager
+  (accessed by pressing `]` in the Julia command prompt):
```
Comment (charleskawczynski) [inline README.md:40]:
```suggestion
  (accessed by pressing `]` in the Julia REPL):
```

**Comment by trontrytel** [comment]
> LGTM.
> 
> It might be nice to point to good Julia documentation/resources on package management since these things can be a bit tricky for newcomers.

Good idea. I added a link to the into in Pkg docs

**Pull request merged**